### PR TITLE
Create chris8889.json

### PR DESCRIPTION
**Pull request by Chris8889**

### CODE REVIEW

**Review by phenax — DISMISSED** [review]
You've added the file in the wrong directory. Please move this to the `domains` directory.

**Comment by Chris8889** [comment]
> You've added the file in the wrong directory. Please move this to the `domains` directory.

I fixed it

File: domains/chris8889.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "testing",
+    "owner": {
+      "username": "Chris",
```
Comment (MaskDuck) [inline domains/chris8889.json:4]: not your username?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/chris8889.json (hunk 1 above)
Comment (Chris8889) [reply domains/chris8889.json]: Oh, I didn't know it had to be my github username. I changed it

**Review by MaskDuck — APPROVED** [review]
okay!

**Pull request merged**